### PR TITLE
Added wizard for subscribing attendees

### PR DESCRIPTION
**Pull request by antonag32**
Completed all exercises under the 'wizard' section.

### CODE REVIEW

**Comment by frahikLV** [comment]
@antonag32 could you rebase this PR?, please

**Comment by antonag32** [comment]
@frahikLV Rebased

File: opa/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+course_read,opa.course.read,model_course,base.group_user,1,0,0,0
+session_read,opa.session.read,model_session,base.group_user,1,0,0,0
+course_all,opa.course.all,model_course,opa.group_manager,1,1,1,1
+session_all,opa.session.all,model_session,opa.group_manager,1,1,1,1
+attendee_wizard_all,opa.attendee.wizard.all,model_attendee_wizard,opa.group_manager,1,1,1,0
```
Comment (frahikLV) [inline opa/security/ir.model.access.csv:6]: @antonag32 Wizards didn't require access rules, if you want that only manager can view it, on menu give access only to that group

**Review by frahikLV — CHANGES_REQUESTED** [review]
@antonag32 can you address this change?

File: opa/security/ir.model.access.csv (hunk 1 above)
Comment (antonag32) [reply opa/security/ir.model.access.csv:6]: If I don't add an access rule I get the following warning:
```
2022-03-28 19:03:59,017 6066 WARNING opatest odoo.modules.loading: The models ['attendee.wizard'] have no access rules in module opa, consider adding some, like:
id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
opa.access_attendee_wizard,access_attendee_wizard,opa.model_attendee_wizard,base.group_user,1,0,0,0 
```
And I am also unable to access the wizard, as I no longer see it on the 'Action' drop down menu. I reviewed some of the original Odoo modules and I also see they add access rules for wizards (for example in `account` for the model `account.automatic.entry.wizard`).
Based on this I think the access rule is necessary.

**Pull request merged**